### PR TITLE
Attendance fixes and changes

### PR DESCRIPTION
**Pull request by roperasher**
-Fixed code to make date ranges work
-Added the header "classStartTime" to report
-New header format:
| classId | classTitle | classStartTime | capacity | registered | attended |
-Fills attendance with dummy value of -9999 if ENOTFOUND error occurs

### CODE REVIEW

File: server/reports/attendance/attendance-endpoint.js [hunk 1]
```diff
@@ -67,47 +61,47 @@ export function attendanceRequestHandler(request, response) {
...
 
       // resolves all, allNumberAttendedPromises then outputs attendance report to endpoint as CSV or JSON
       Promise.all(allNumberAttendedPromises)
         .then(() => {
           if (format === "csv") {
-            let fields = ["class", "capacity", "registered", "attended"];
+            let fields = ["classId", "classTitle", "classStartTime", "capacity", "registered", "attended"];
             let parser = new J2C.Parser({ fields });
             let csv = parser.parse(attendanceReport);
             let fileName = "AttendanceReport.csv";
```
Comment (DanielBrewerPsu) [inline server/reports/attendance/attendance-endpoint.js:107]: CSV download doesn't work for me and always gives me and empty report. Does it work for you? ll push one way to fix it in a attendance-fixes-and-changes_fix1 branch.

Comment (roperasher) [reply server/reports/attendance/attendance-endpoint.js:107]: I've never had that problem. I've always gotten back a csv with a full report.

Comment (roperasher) [reply server/reports/attendance/attendance-endpoint.js:107]: When you say "CSV download doesn't work for me and always gives me an empty report" do you mean the download never happens and nothing is output to the endpoint, or the download happens, but nothing is in the .csv file?

Comment (roperasher) [reply server/reports/attendance/attendance-endpoint.js:107]: Hmm... Sorry I just tried again and I got a CSV output with no data in it. I have a ton of CSV files that I got back with data in it, so I don't know at what point this stopped working. It looks like you were able to fix it in PR #50. Thank you for catching that.

**Pull request merged**